roles/ipaserver/library/ipaserver_test.py

```diff
@@ -953,51 +953,36 @@ def main():
 
     #########################################################################
 
-    http_pkcs12_file = None
     http_pkcs12_info = None
     http_ca_cert = None
-    dirsrv_pkcs12_file = None
     dirsrv_pkcs12_info = None
     dirsrv_ca_cert = None
-    pkinit_pkcs12_file = None
     pkinit_pkcs12_info = None
     pkinit_ca_cert = None
 
     if options.http_cert_files:
         if options.http_pin is None:
             ansible_module.fail_json(
                 msg="Apache Server private key unlock password required")
-        http_pkcs12_file, http_pin, http_ca_cert = load_pkcs12(
-            cert_files=options.http_cert_files,
-            key_password=options.http_pin,
-            key_nickname=options.http_cert_name,
-            ca_cert_files=options.ca_cert_files,
-            host_name=host_name)
-        http_pkcs12_info = (http_pkcs12_file.name, http_pin)
+        http_pkcs12_info = [options.http_cert_files[0], options.http_pin]
```

Comment:
This is removing the tests of the certificate. I think it is needed to copy the generated local temporary files (they are automatically removed when they are closed) and to return the paths in ipaserver_test.